test/libsolidity/syntaxTests/inheritance/base_arguments_empty_parentheses.sol

```diff
@@ -0,0 +1,5 @@
+contract Base {
+  function Base(uint) public {}
+}
+contract Derived is Base(2) { }
+contract Derived2 is Base(), Derived() { }
```

Comment:
Can you add `contract Derived3 is Base, Derived {}` to make sure the syntactical difference wont cause any issue down the line?